frontend/src/app/actions/newsletter-form.js

```diff
@@ -0,0 +1,29 @@
+import { createActions } from 'redux-actions';
+
+const actions = createActions(
+  {
+    newsletterFormSetEmail: email => email,
+    newsletterFormSetPrivacy: privacy => privacy
+  },
+  'newsletterFormSetFailed',
+  'newsletterFormSetSubmitting',
+  'newsletterFormSetSucceeded'
+);
+
+const subscribeActions = createActions({
+  newsletterFormSubscribe: (dispatch, email) => {
```

Comment:
@lmorchard so, I think it does make sense for actions to dispatch other actions; this pattern is even featured in Redux's docs: http://redux.js.org/docs/advanced/AsyncActions.html#async-action-creators

The problem is in the `redux-actions` library, which cuts some boilerplate at the cost of functionality.